code/__DEFINES/obj_flags.dm

```diff
@@ -54,6 +54,7 @@
 #define ANTI_TINFOIL_MANEUVER   (1<<12) //Hats with negative effects when worn (i.e the tinfoil hat).
 #define DANGEROUS_OBJECT    (1<<13)     //Clothes that cause a larger notification when placed on a person.
 #define LARGE_WORN_ICON    (1<<14)     //Clothes that use large icons, for applying the proper overlays like blood
+#define BLOCKS_SPEECH (1<<15) // Clothes that block speech (i.e the muzzle). Can be applied to any clothing piece.
```

Comment:
```suggestion
/// Clothes that block speech (i.e the muzzle). Can be applied to any clothing piece.
#define BLOCKS_SPEECH (1<<15)
```

Doc comment.

Reply:
If you say so, it'd break the standard format of the document.

Reply:
The rest of the document should be changed at some point, like I did to traits.